src/ui/ui-kit/BottomPanel/styles.module.css

```diff
@@ -0,0 +1,24 @@
+.stickyBottomSheet {
+  position: sticky;
+  bottom: 0;
+  padding-bottom: 16px;
+}
+
+.content {
+  pointer-events: all;
```

Comment:
Why is this needed?

Reply:
Forgot to remove after removing animated version